create_batch.py

```diff
@@ -57,11 +57,12 @@ def parse_arguments(args):
     return job_submission_params
 
 
-def get_job_file_lines(job_name, job_file_name, scheduler=None, memory=3600, walltime="4:00", queue=None):
+def get_job_file_lines(job_name, job_file_name, email_notif, scheduler=None, memory=3600, walltime="4:00", queue=None):
```

Comment:
What's the reason to default `email_notif` to `False` in some cases but not all? My thought is the desired functionality would be to always default to `False`.

Reply:
@2gotgrossman I wrote it to default to `False` when submitting a batch of jobs, but to `True` when submitting major scripts based on what Falk had said in the issue ("For create_batch.py the default should be WITHOUT email notification, for process_rsmas the default should be WITH email notification"). But that could be wrong if it is supposed to only be `True` for `process_rsmas.py` and `False` for other major scripts